cargo-dist/src/tasks.rs

```diff
@@ -2428,17 +2434,112 @@ pub(crate) struct AnnouncementTag {
     pub rust_releases: Vec<(PackageIdx, Vec<String>)>,
 }
 
+/// details on what we're announcing (partially computed)
```

Comment:
ah this might answer my question from above. to check my understanding- partially computer means some info is potentially provided by the user where others are coming from compute we're doing?

Reply:
Yeah basically. Notably there's more Option/missing data that is guaranteed to exist by the end.